src/morphAttrs.js

```diff
@@ -16,9 +16,15 @@ export default function morphAttrs(fromNode, toNode) {
         attrValue = attr.value;
 
         if (attrNamespaceURI) {
-            attrName = attr.localName || attrName;
-            fromValue = fromNode.getAttributeNS(attrNamespaceURI, attrName);
-
+            var attrLocalName = attr.localName;
+
+            // Important: getAttributeNS expects the localName of a namespaced attribute
+            // but setAttributeNS requires the fully qualified name
+            // ref: https://dom.spec.whatwg.org/#dom-element-getattributens
+            // ref: https://www.w3.org/TR/DOM-Level-2-Core/glossary.html#dt-localname
+            // ref: https://dom.spec.whatwg.org/#dom-element-setattributens
+            // ref: https://www.w3.org/TR/DOM-Level-2-Core/glossary.html#dt-qualifiedname
+            fromValue = fromNode.getAttributeNS(attrNamespaceURI, attrLocalName || attrName);
```

Comment:
@AutoSponge 
I just inlined the attrName fallback just for `getAttributeNS`.

Reply:
@AndyOGo are you saying these two blocks are different?

current

```js
attrName = attr.localName || attrName;
fromValue = fromNode.getAttributeNS(attrNamespaceURI, attrName);
```
this PR
```js
var attrLocalName = attr.localName;
fromValue = fromNode.getAttributeNS(attrNamespaceURI, attrLocalName || attrName);
```

Because I'm not seeing it. I'm still confused as to exactly what you think this fixes.

Reply:
No with my last commit I restored the original `getAttributeNS` function call.

But the line below using `setAttributeNS` has changed.